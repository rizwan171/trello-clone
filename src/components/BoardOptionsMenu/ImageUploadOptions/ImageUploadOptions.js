import React from "react";
import { AiOutlinePicture, AiOutlineSave } from "react-icons/ai";

const ImageUploadOptions = () => {
  return (
    <div className="flex flex-col">
      <div className="border-dashed border-3 border-trello-blue-100 h-38 flex flex-col items-center justify-center cursor-pointer">
        <AiOutlinePicture size={75} className="text-gray-600" />
        <p className="text-xs text-gray-600">Click here to Upload a picture</p>
      </div>
      <button className="flex justify-center items-center py-2 px-3 mt-2 bg-trello-blue-100 hover:bg-trello-blue-200 text-white text-base shadow-md rounded-md">
        <AiOutlineSave size={23} className="mr-2" />
        Save as Background
      </button>
    </div>
  );
};

export default ImageUploadOptions;
