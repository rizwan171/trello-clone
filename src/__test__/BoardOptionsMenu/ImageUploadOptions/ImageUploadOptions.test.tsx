import TestRenderer from "react-test-renderer";
import ImageUploadOptions from "../../../components/BoardOptionsMenu/ImageUploadOptions/ImageUploadOptions";

describe("ImageUploadOptions", () => {
  it("should render successfully", () => {
    const component = TestRenderer.create(<ImageUploadOptions />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
