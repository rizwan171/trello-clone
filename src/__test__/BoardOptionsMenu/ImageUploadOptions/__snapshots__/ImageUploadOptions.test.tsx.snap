// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageUploadOptions should render successfully 1`] = `
<div
  className="flex flex-col"
>
  <div
    className="border-dashed border-3 border-trello-blue-100 h-38 flex flex-col items-center justify-center cursor-pointer"
  >
    <svg
      className="text-gray-600"
      fill="currentColor"
      height={75}
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 1024 1024"
      width={75}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M928 160H96c-17.7 0-32 14.3-32 32v640c0 17.7 14.3 32 32 32h832c17.7 0 32-14.3 32-32V192c0-17.7-14.3-32-32-32zm-40 632H136v-39.9l138.5-164.3 150.1 178L658.1 489 888 761.6V792zm0-129.8L664.2 396.8c-3.2-3.8-9-3.8-12.2 0L424.6 666.4l-144-170.7c-3.2-3.8-9-3.8-12.2 0L136 652.7V232h752v430.2zM304 456a88 88 0 1 0 0-176 88 88 0 0 0 0 176zm0-116c15.5 0 28 12.5 28 28s-12.5 28-28 28-28-12.5-28-28 12.5-28 28-28z"
      />
    </svg>
    <p
      className="text-xs text-gray-600"
    >
      Click here to Upload a picture
    </p>
  </div>
  <button
    className="flex justify-center items-center py-2 px-3 mt-2 bg-trello-blue-100 hover:bg-trello-blue-200 text-white text-base shadow-md rounded-md"
  >
    <svg
      className="mr-2"
      fill="currentColor"
      height={23}
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 1024 1024"
      width={23}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M893.3 293.3L730.7 130.7c-7.5-7.5-16.7-13-26.7-16V112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V338.5c0-17-6.7-33.2-18.7-45.2zM384 184h256v104H384V184zm456 656H184V184h136v136c0 17.7 14.3 32 32 32h320c17.7 0 32-14.3 32-32V205.8l136 136V840zM512 442c-79.5 0-144 64.5-144 144s64.5 144 144 144 144-64.5 144-144-64.5-144-144-144zm0 224c-44.2 0-80-35.8-80-80s35.8-80 80-80 80 35.8 80 80-35.8 80-80 80z"
      />
    </svg>
    Save as Background
  </button>
</div>
`;
